### ISSUE TITLE
[FEATURE] I18N management

### ISSUE THREAD

**Issue by ncorre** [Baboo7/strapi-plugin-import-export-entries#43]
![export](https://user-images.githubusercontent.com/2435203/185485544-e03a46d2-5df8-45da-a48a-de31d4ecdc74.png)
Hi,
I'm currently using your plugin and I would like to know if this case is possible :

- I have 2 locales in my project (FR as default and EN)
- When I export a single type by clicking on the export button, I can all the stuff (FR and EN)
- For my example I'm deleting the content of my single types and I do an import of my JSON file
- Only the FR is pre populated, EN is still empty.

here is an example of my export JSON :
```
{
	"id": 5,
	"about_hero_title": "Notre Histoire",
	"about_hero_desc": "Capitalisant sur une expérience multifonctionnelle en planification couplée à une forte appétence technique, nous avons pris conscience de l’importance grandissante de profils pluridisciplinaires dans les entreprises.<br><br>\n\nNotre esprit entrepreneurial nous a conduit à la création de OneHive avec l’ambition de construire une communauté d’experts pour progresser ensemble autour de la mise en valeur opérationnelle des données.<br><br>",
	"about_hero_desc_2": "Anciens utilisateurs opérationnels d’ANAPLAN, solution cloud pionnier du « Connected Planning », nous sommes convaincus par la pertinence de l’outil pour adresser l’ensemble des sujets de planification.",
	"createdAt": "2022-08-17T02:07:47.122Z",
	"updatedAt": "2022-08-18T19:47:50.269Z",
	"publishedAt": "2022-08-17T02:15:48.065Z",
	"locale": "fr",
	"about_hero_logo": {
		"id": 4,
		"name": "Anaplan.png",
		"alternativeText": "Anaplan.png",
		"caption": "Anaplan.png",
		"width": 227,
		"height": 31,
		"formats": null,
		"hash": "Anaplan_b1fbb4fdd6",
		"ext": ".png",
		"mime": "image/png",
		"size": 1.33,
		"url": "/uploads/Anaplan_b1fbb4fdd6.png",
		"previewUrl": null,
		"provider": "local",
		"provider_metadata": null,
		"folderPath": "/4",
		"createdAt": "2022-08-17T02:07:27.814Z",
		"updatedAt": "2022-08-17T02:07:27.814Z"
	},
	"createdBy": null,
	"updatedBy": null,
	"localizations": [
		{
			"id": 6,
			"about_hero_title": "Our story",
			"about_hero_desc": "Capitalizing on both multifunctional experiences in planning and valuable technical abilities, we strongly believe in the power of multidisciplinary profiles in companies.<br><br>\n\nOur entrepreneurial spirit has led us to OneHive's founding, bearing the ambition of building an expert community to grow and move forward together in the enhancement of operational data.  <br><br>",
			"about_hero_desc_2": "Former business users of ANAPLAN, the leading cloud-based platform of \"Connected Planning\", we are deeply convinced by the tool to address all planning topics.",
			"createdAt": "2022-08-18T19:47:50.257Z",
			"updatedAt": "2022-08-18T19:47:50.257Z",
			"publishedAt": null,
			"locale": "en"
		}
	]
}
```
Is it beauce I'm doing something wrong or it's because the plugin is not made for I18N usage ?

Thanks for the work and help.
Nic

**Comment by Baboo7** [comment]
I can't lie, i18n support needs some love ❤️ You're doing things right, I need to dig further how i18n plugin works to have it working as expected

**Comment by Baboo7** [comment]
Hey @ncorre, after some important refacto I just published version `1.16.0` where export/import of locales is supported for `JSON v2` format 🎉 I would love to have your feedback on this new version
